dojo_plugin/models/__init__.py

```diff
@@ -58,6 +58,7 @@ class Dojos(db.Model):
     id = db.Column(db.String(32), index=True)
     name = db.Column(db.String(128))
     description = db.Column(db.Text)
+    icon = db.Column(db.String(256))
```

Comment:
JSON is way more flexible to update/change. I know it's "dirty" but the flexibility makes it worth it. We make fields not-JSON if they're absolutely all over the code base/we need to be able to filter and sort based on it at the database level (for performance reasons).